pkg/webhook/server/server.go

```diff
@@ -62,7 +62,7 @@ func (s *WebhookServer) Start() error {
 		tlsConf.ClientCAs = roots
 	}
 
-	listenAddr := app.ValidatingWebhookSettings.ListenAddr + ":" + app.ValidatingWebhookSettings.ListenPort
+	listenAddr := s.Settings.ListenAddr + ":" + s.Settings.ListenPort
```

Comment:
net package contains precious function `JoinHostPort` for this purpose :)

Reply:
Done. JoinHostPort is used now.